src/main/io/msp.c

```diff
@@ -942,12 +942,8 @@ static int processOutCommand(mspPacket_t *cmd, mspPacket_t *reply)
 
         case MSP_LED_STRIP_CONFIG:
             for (int i = 0; i < LED_MAX_STRIP_LENGTH; i++) {
-                ledConfig_t *ledConfig = ledConfigs(i);
```

Comment:
@ledvinap ; I like the simplicity and compacity of sending the raw uint32_t of each LED over MSP.
It saves memory too, which was one of the goals of the commit.
A compatible Configurator update is ready.
So I'm keeping this unchanged. If you feel differently I am open to discuss this further of course.